manifests/l/LX-Systems/WinMute/2.1.2/LX-Systems.WinMute.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: LX-Systems.WinMute
+PackageVersion: 2.1.2
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
Application only installs on x64 OS.